test/on_yubikey/test_piv.py

```diff
@@ -0,0 +1,121 @@
+import unittest
+from binascii import a2b_hex
+from ykman.driver_ccid import APDUError
+from ykman.piv import PivController
+from ykman.util import TRANSPORT
+from .util import (DestructiveYubikeyTestCase, missing_mode, open_device)
+
+
+DEFAULT_PIN = '123456'
+NON_DEFAULT_PIN = '654321'
+DEFAULT_PUK = '12345678'
+NON_DEFAULT_PUK = '87654321'
+DEFAULT_MANAGEMENT_KEY = '010203040506070801020304050607080102030405060708'
+NON_DEFAULT_MANAGEMENT_KEY = '010103040506070801020304050607080102030405060708'
+
+
+@unittest.skipIf(*missing_mode(TRANSPORT.CCID))
+class PivTestCase(DestructiveYubikeyTestCase):
+    pass
+
+
+class ManagementKey(PivTestCase):
+
+    def setUp(self):
+        self.dev = open_device(transports=TRANSPORT.CCID)
+        self.ctrl = PivController(self.dev.driver)
```

Comment:
Change to `self.controller`. Not because I think the short version is bad, but for consistency.

Reply:
Sure. I didn't think much of the naming since this is just a self-contained test, but that's a very good point.